### DISCUSSION TITLE
What did you use to generate tables for the benchmarks?

### DISCUSSION THREAD

**Discussion by etcd** [streamich/json-joy#596] [Q&A] [answered]
Hey, not sure where better to ask this question, but I'd love to know how you made [these benchmark tables](https://jsonjoy.com/blog/list-crdt-benchmarks)!

**Comment by streamich** [comment, marked as answer]
For data: scripts in `/src/json-crdt/__bench__`. For images: Google Spreadsheets + Figma :)
